playbooks/automember/automember-present.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Automember present example
+  hosts: ipaserver
+  become: true
+  tasks:
+  - name: Ensure group automember rule admins is present
+    ipaautomember:
+      ipaadmin_password: SomeADMINpassword
+      name: admins
+      type: group
+      state: present
+
+  - name: Ensure hostgroup automember rule ipaservers is absent
```

Comment:
Please, separate this playbook in two, one for group, one for hostgroup.

Reply:
Done